test/Directory.Build.targets

```diff
@@ -1,5 +1,13 @@
 <Project>
 
   <Import Project="..\Directory.Build.targets" />
-
+  
+  <PropertyGroup Condition="'$(Coverage)' == 'true' and '$(Configuration)'=='Release' and '$(OS)' == 'Windows_NT' and '$(TargetArchitecture)' == 'x64'">
+      <CollectCoverage>true</CollectCoverage>
+      <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
Why is this necessary? Does the coverage tool require it? If so, can you link to information on it?